.circleci/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
+- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
+- [ ] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)
+
+### What this does
+
+_Explain why this PR exists_
+
+### Notes for the reviewer
+
+_Instructions on how to run this locally, background context, what to review, questions…_
+
+### More information
+
+- [Link to documentation](https://github.com/snyk/snyk-api-ts-client/wiki/)
```

Comment:
change the repo name to `jira-tickets-for-new-vulns`

Reply:
Done